packages/build/tests/plugins/fixtures/engines/package.json

```diff
@@ -5,6 +5,6 @@
   "license": "MIT",
   "repository": "test",
   "engines": {
-    "node": ">=10.0.0"
```

Comment:
We bumped the version in our `engines` test because we needed something that's greater than `10.18.0` (in order to not get caught in our minimum version conditions).